Cargo.toml

```diff
@@ -11,22 +11,22 @@ readme = "README.md"
 [dependencies]
 thiserror = "1"
 bs58 = {version = "0.4", features=["check"]}
-base64 = "0"
+base64 = "0.21"
```

Comment:
can you keep this at "0", please? It allows downstream to push for a higher version.

Reply:
I'm not sure that's super desirable; Rust semver is a little different from canonical semver, and rust considers any minor version bump a breaking change while the major version is 0. So if downstream is left to choose the latest version, it can cause this crate to fail compilation

Reply:
The problem is that some sibling chooses a different version